### ISSUE TITLE
Usage of taskkill.exe on linux

### ISSUE THREAD

**Issue by davyv-equans** [profesorfalken/jPowerShell#70]
https://github.com/profesorfalken/jPowerShell/blob/fe785459bca019e6d0dd2dda7cc7ee4074219592/src/main/java/com/profesorfalken/jpowershell/PowerShell.java#L407

The lib is not completely converted to work in linux. When closing the powershell there is still hardcoded the call to taskkill.exe, which triggers an exception.

**Comment by jenskreidler** [comment]
Yes, one could implement the unix part: like "OSDetector" ... then for unix "kill -9 pid".
